docs/book/src/user/quick-start.md

```diff
@@ -1179,6 +1185,3 @@ spec:
 
 {{#/tab }}
 {{#/tabs }}
-
-<!-- links -->
-[target cluster]: ../reference/glossary.md#target-cluster
```

Comment:
Decided to remove this link for now to reduce confusion until we have proper Glossary terminology. 
See this issue/comment for more details: https://github.com/kubernetes-sigs/cluster-api/pull/2042#issuecomment-576767956